packages/react-sdk/src/state/communication/matrixRtcCommunicationChannel.ts

```diff
@@ -257,7 +310,7 @@ export class MatrixRtcCommunicationChannel implements CommunicationChannel {
 
     // Wait for the Widget API and LiveKit servers to be ready
     await this.widgetApiPromise;
-    await this.initLiveKitServer(session);
+    await this.initLiveKitBackend();
```

Comment:
I think we should check if the current `handleSessionJoined` needs to be invoked when another user joins. It doesn't look so.